contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down, which stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
+
+PostHog started off as a completely flat company with one big goal: to increase the number of successful products in the world.
+
+As we are getting bigger, we realized that it's harder for people to see the direct impact of their work, which reduces the sense of ownership.
+
+We have therefore introduced Small Teams. These are designed to each operate like a startup. 
+
+# How it works
+
+* A Small Team should never be more than ten people
+* A Small Team has an Accountable Person responsible for its performance - whoever is most appropriate depending on what the team is working on. Who would make the right founder for that bit of the company?
+* A Small Team must have (1) a customer, (2) a mission and (3) metrics
```

Comment:
A customer might be internal, e.g. the team

Reply:
While we are on this, I would suggest another look into OKRs in order to track long-term metrics, rather than week on week (like we currently do with retention for example)

Reply:
> While we are on this, I would suggest another look into OKRs in order to track long-term metrics, rather than week on week (like we currently do with retention for example)

[Started this issue here](https://github.com/PostHog/ops/issues/194) to discuss - I want to work on it in the next sprint.